pkg/virt-operator/resource/apply/instancetype_test.go

```diff
@@ -95,11 +105,16 @@ var _ = Describe("Apply Instancetypes", func() {
 			}),
 		)
 
-		It("should delete all instancetypes managed by virt-operator", func() {
+		It("should delete all deployed instance types managed by virt-operator", func() {
+			reconciler.stores.ClusterInstancetype.Add(instancetype)
 			deleteCollectionCalled := expectDeleteCollection(fakeClient, reconciler.kv, apiinstancetype.ClusterPluralResourceName)
 			Expect(reconciler.deleteInstancetypes()).To(Succeed())
 			Expect(*deleteCollectionCalled).To(BeTrue())
 		})
+
+		It("should not call delete if no instance types have been deployed by virt-operator", func() {
+			Expect(reconciler.deleteInstancetypes()).To(Succeed())
```

Comment:
How does this assert delete was not called?

Reply:
L34 asserts that no unexpected calls were made by the client.